proto/bragi/common.proto

```diff
@@ -49,25 +49,33 @@ message ControlErrorAnnouncement {
   optional string message = 2;
 }
 
+message MatchTimeline {
+  repeated Match matches = 1;
+}
+
 message Match {
   string match_urn = 1;
-  string tournament_urn = 2;
+  string tournament_urn = 2 [deprecated=true]; // Deprecated. Its being returned on the tournament object
   Sport sport = 3;
-  string home_team_urn = 4;
-  string away_team_urn = 5;
+  string home_team_urn = 4 [deprecated=true]; // Deprecated. Its being returned on the competitor object where home_team will be true
+  string away_team_urn = 5 [deprecated=true]; // Deprecated. Its being returned on the competitor object where home_team will be false
   uint32 home_score = 6;
   uint32 away_score = 7;
   MatchStatusType match_status = 8;
   google.protobuf.Timestamp plannet_start = 9;
+  Tournament tournament  = 10;
+  repeated Competitor competitors = 11;
 }
 
-message MatchTimeline {
-  repeated Match matches = 1;
+message Tournament {
+  string id = 1;
+  string name = 2;
 }
 
-message Team {
-  string team_urn = 1;
-  string team_name = 2;
+message Competitor {
+  string id = 1; // id is urn in format e.g. 'od:team:123' or 'od:player:123'
+  string name = 2;
+  optional bool home_team = 3;
```

Comment:
do we want to use it only for home team? Can we use enum as HOME, AWAY and then we can add something else in future if new sport will require it.

Reply:
Good idea. Changed